distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1392,6 +1392,76 @@ type Table
         msg = "Parsing values is not supported in database tables, the table has to be materialized first with `read`."
         Error.throw (Unsupported_Database_Operation.Error msg)
 
+    ## Splits a column of text into a set of new columns.
+       The original column will be removed from the table.
+       The new columns will be named with the name of the input with a
+       incrementing number after.
+
+       Arguments:
+       - column: The column to split the text of.
```

Comment:
```suggestion
       - column: The column name or index to split the text of.
```

I think we need to say that we are looking for name/index here, **not** a _column reference_.